src/Agent.Worker/Container/DockerCommandManager.cs

```diff
@@ -187,11 +189,11 @@ public async Task<List<string>> DockerPS(IExecutionContext context, string optio
 
         public async Task<int> DockerNetworkCreate(IExecutionContext context, string network)
         {
-#if OS_WINDOWS
-            return await ExecuteDockerCommandAsync(context, "network", $"create --label {DockerInstanceLabel} {network} --driver nat", context.CancellationToken);
-#else
+// #if OS_WINDOWS
```

Comment:
can you remove these commented lines now?

Reply:
I forgot about this and need to figure out what the right thing to do here is